Sources/Tracking/CustomerIO.swift

```diff
@@ -299,6 +360,10 @@ public class CustomerIO: CustomerIOInstance {
         implementation?.identify(identifier: identifier, body: body)
     }
 
+    public func registeredDeviceToken() -> String? {
```

Comment:
🚧  

Reminder to add code to the `Tracking/APITest` class for this new public API change.

Reply:
Let's decide on [this](https://github.com/customerio/customerio-ios/pull/378#discussion_r1314538993) and then go write test cases. As mentioned in the description, test cases is a part of pending list, hence a draft PR.